### PR TITLE
3step using signed s3 url

### PR DESCRIPTION
**Pull request by jackneil**
adding capabilities to handle presigned url 3-step posting of documents to our api

work in progress

### CODE REVIEW

**Review by techdozor — COMMENTED** [review]
Can't review until I see how it will be implemented. Especially I am not sure if thats entirely valid how post will be handled. 
I would let the implementation to be completed.

**Comment by techdozor** [comment]
We will likely introduce one more request syntax TBD to obtain th URI that later will go as part of the POST in order to avoid the change to the rest of the core architecture that spans varity of APIs but we will make sure to update this library as it comes to fruition.

**Comment by jackneil** [comment]
Understood. I'm not writing any more code on there until serverside implmenetation is done but went ahead and modified it to get close while it was fresh in my head.

Didn't intend for it to be reviewed yet just pushed the commit to the branch to back it up from my computer.

Jack Neil
CEO | Hank AI, Inc.
***@***.***
(w) +1.904.GET.HANK
(m) +1.803.466.7551
711 East Main Street, Suite E, Lexington, SC 29072

Sent via Superhuman ( ***@***.*** )

**Comment by techdozor** [comment]
👍

**Review by techdozor — APPROVED** [review]
looks good

File: docuvision/batchsendfulldir.py [hunk 1]
```diff
@@ -75,72 +86,114 @@
...
+        files = {'file': dv_file['base64encoded']}
+        resp = None
+        resp = requests.post(
+            url=presignedurl_details.get('url'), #docuvision/v1/tasks/7",
+            data=presignedurl_details.get('fields'),
+            files=files,
+            timeout=timeout
+        )
+        if resp.status_code<300:
+            return 1
```
Comment (thorstt) [inline docuvision/batchsendfulldir.py:124]: Given how these are used, it more idiomatic to return True or False.

**Review by thorstt — APPROVED** [review]
Looks ok.

**Pull request merged**